target/linux/realtek/files-5.10/drivers/clk/realtek/Kconfig

```
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: GPL-2.0-only
+config COMMON_CLK_RTL83XX
+	bool "Clock driver for Realtek RTL83XX"
+	depends on RTL83XX
+	select SRAM
+	help
+	  This driver supports Realtek RTL83XX basic clocks.
```

checkpatch usually complains (or used to) about very short help entries. I know; adding words to please the linter is usually silly.

```suggestion
	  This driver adds support for the Realtek RTL83xx series basic clocks. This includes chips in the
	  RTL838x series, such as RTL8380, RTL8381, RTL832, as well as chips from the RTL839x series, such as RTL8390, RT8391, RTL8392, RTL8393 and RTL8396.
	  
	  When configured as a module, the module will be called clk-rtl83xx.
```

this enumerations are really useful for people going through the 'help' and trying to figure out what to enable :)